### PR TITLE
feat: conditional compilation for wasm runtimes

### PR DESCRIPTION
**Pull request by wujunzhuo**

### CODE REVIEW

File: serverless/wasm/wasmtime.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//go:build wasmtime
```
Comment (venjiang) [inline serverless/wasm/wasmtime.go:1]: wasmtime as the default, no conditional compilation required, what do you think?

Comment (wujunzhuo) [reply serverless/wasm/wasmtime.go:1]: this allows users to compile with **OR** without wasmtime

**Review by venjiang — APPROVED** [review]
Good job! 👍

**Review by fanweixiao — APPROVED** [review]

**Comment by fanweixiao** [comment]
🥳🥳

**Pull request merged**